pkg/hintrunner/core/hint.go

```diff
@@ -1237,7 +1238,8 @@ func (hint *InitSquashData) Execute(vm *VM.VirtualMachine, ctx *hinter.HintRunne
 	// todo(rodro): Don't know if it could be called multiple times, or
 	err := hinter.InitializeSquashedDictionaryManager(ctx)
 	if err != nil {
-		return err
+		ctx.SquashedDictionaryManager = hinter.SquashedDictionaryManager{}
+		_ = hinter.InitializeSquashedDictionaryManager(ctx)
```

Comment:
I assume this means you can call this method even if the squashed dict has already been initialized in the past, right? Shouldn't the `todo` above be removed then?

Reply:
this should overwrite the old manager. i agree that we can remove the todo